.prettierrc.json

```diff
@@ -1,3 +1,30 @@
 {
-  "parser": "html"
+  "semi": true,
+  "singleQuote": true,
+  "tabWidth": 2,
+  "useTabs": true,
+  "trailingComma": "es5",
+  "bracketSpacing": true,
+  "arrowParens": "always",
+  "printWidth": 80,
+  "overrides": [
+    {
+      "files": "*.html",
+      "options": {
+        "parser": "html"
+      }
+    },
+    {
+      "files": "*.js",
+      "options": {
+        "parser": "babel"
+      }
+    },
+    {
+      "files": "*.njk",
```

Comment:
I think the way things are written now the CI won't pass .njk files to prettier; am I missing something?

Reply:
Removed the .njk parser override that was causing issues with .njk files.